semantic/label_set.go

```diff
@@ -35,92 +35,3 @@ func (s LabelSet) String() string {
 func (s LabelSet) isAllLabels() bool {
```

Comment:
Is label_set used anywhere now? It's purpose was soley used for implementing type inference in Go. I expect it is unused at this point.

Reply:
It's still used in `values/object.go`, but only there.  I'll move it and make it private.